docs/manager/graphql-reference/schema.graphql

```diff
@@ -570,6 +573,9 @@ type AgentNode implements Node {
   auto_terminate_abusing_kernel: Boolean
   local_config: JSONString
   container_count: Int
+
+  """Added in 25.2.0."""
+  gpu_alloc_map: JSONString
```

Comment:
Is it possible to define the actual type instead of JSONString?

Reply:
I think unless we define a custom scalar type (like `UUIDFloatMap`), there will be no other way.

Reply:
I added `UUIDFloatMap ` in #3293